docs/examples/js/secure_cell.js

```diff
@@ -1,18 +1,66 @@
-var addon = require('jsthemis');
-
-
-if(process.argv.length==5){
-    command=process.argv[2];
-    password=new Buffer.from(process.argv[3]);
-
-    seal = new addon.SecureCellSeal(password);
-    if(command=="enc"){
-	console.log(seal.encrypt(new Buffer.from(process.argv[4])).toString("base64"));
-    }else if(command=="dec"){
-	console.log(seal.decrypt(new Buffer.from(process.argv[4], "base64")).toString());
-    }else{
-	console.log("usage node secure_cell.js <enc/dec> <password> <message>");
-    }
-}else{
-    console.log("usage node secure_cell.js <enc/dec> <password> <message>");
-}
\ No newline at end of file
+const themis = require('jsthemis')
+
+let message = 'Test Message Please Ignore'
+let context = 'Secure Cell example code'
+let master_key = Buffer.from('bm8sIHRoaXMgaXMgbm90IGEgdmFsaWQgbWFzdGVyIGtleQ==', 'base64')
```

Comment:
🤣 

<details><summary>lol</summary>
<p>no, this is not a valid master key</p>
</details>